filebeat/README.md

```diff
@@ -29,4 +30,21 @@ When you run `datadog-agent info` you should see something like the following:
 
 ## Compatibility
 
-The filebeat check is compatible with all major platforms
+The Filebeat check is compatible with all major platforms
```

Comment:
Missing a period (looks to be the case throughout the PR).

Reply:
👍